src/apps/ociocheck/main.cpp

```diff
@@ -209,6 +209,40 @@ int main(int argc, const char **argv)
             for(int i=0; i<config->getNumLooks(); ++i)
             {
                 std::cout << config->getLookNameByIndex(i) << std::endl;
+
+                OCIO::ConstLookRcPtr look = config->getLook(config->getLookNameByIndex(i)); 
+
+                OCIO::ConstTransformRcPtr transform = look->getTransform();         
+                OCIO::ConstTransformRcPtr invTransform = look->getInverseTransform();         
+
+                if(transform!=NULL)
```

Comment:
[Coding Style] The std::shared_ptr supports [boolean cast](en.cppreference.com/w/cpp/memory/shared_ptr/operator_bool) to check if null or not. 
So the code can be simplified like `if (transform)`